### PR TITLE
Homework38

### PR DESCRIPTION
**Pull request by cinck**
Hi!
Please review my homework.

### CODE REVIEW

File: djangoProject/book/models.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+    class Meta:
+        db_table = 'books'
+        unique_together = ['title', 'author']   # <HW37> Task 5. Unique title-author combination
+        verbose_name = 'Book'
+        verbose_name_plural = 'Books'
+
+    def __str__(self):
+        return f"{self.id}: {self.title} ({self.author}, {self.year})"
+        pass
```
Comment (acman) [inline djangoProject/book/models.py:20]: nit: pass is redundant there

File: djangoProject/purchase/admin.py [hunk 2]
```diff
@@ -0,0 +1,9 @@
+from django.contrib import admin
+from purchase.models import Purchase
+# Register your models here.
+
+
+@admin.register(Purchase)
+class PurchaseAdmin(admin.ModelAdmin):
+    list_display = ("pk", "date", "user", "book")
+    pass
```
Comment (acman) [inline djangoProject/purchase/admin.py:9]: same

File: djangoProject/users/admin.py [hunk 3]
```diff
@@ -0,0 +1,9 @@
+from django.contrib import admin
+from django.contrib.auth import get_user_model
+from django.contrib.auth.admin import UserAdmin
+
+
+@admin.register(get_user_model())
+class CustomUserAdmin(UserAdmin):
```
Comment (acman) [inline djangoProject/users/admin.py:7]: 👍

File: djangoProject/users/admin.py [hunk 4]
```diff
@@ -0,0 +1,9 @@
...
+    list_display = ("id", "username", "first_name", "last_name", "email", "age", "is_staff")
+    pass
```
Comment (acman) [inline djangoProject/users/admin.py:9]: same. `pass` is used to make function and class empty

**Review by acman — APPROVED** [review]
GJ!

File: djangoProject/users/admin.py (hunk 4 above)
Comment (cinck) [reply djangoProject/users/admin.py:9]: I know. It's just my code writing feature. I create empty function or class with 'pass' item. Then I write their functionality and don't remove 'pass' statement, if I plan to delete and rewrite something totally in future.

**Pull request merged**